src/Microsoft.Dnx.Runtime.Internals/RuntimeEnvironmentExtensions.cs

```diff
@@ -31,8 +32,64 @@ public static string GetShortVersion(this IRuntimeEnvironment env)
             return $"{env.RuntimeType}-{env.RuntimeArchitecture}-{env.RuntimeVersion}";
         }
 
+        public static IEnumerable<string> GetDefaultRestoreRuntimes(this IRuntimeEnvironment env)
```

Comment:
This needs to go into the tooling project. I don't want anything from restore leaking into shared assets.